scripts/build_biomass_potentials.py

```diff
@@ -57,7 +58,12 @@ def build_biomass_potentials():
         snakemake.input['jrc_potentials'] = "data/biomass/JRC Biomass Potentials.xlsx"
         snakemake.output = Dict()
         snakemake.output['biomass_potentials'] = 'data/biomass_potentials.csv'
+        snakemake.output['biomass_potentials_all']='resources/biomass_potentials_all.csv'
         with open('config.yaml', encoding='utf8') as f:
             snakemake.config = yaml.safe_load(f)
-
+    
+    if 'Secondary Forestry residues sawdust' in snakemake.config['biomass']['classes']['solid biomass']:
+        snakemake.config['biomass']['classes']['solid biomass'].remove('Secondary Forestry residues sawdust')
+        snakemake.config['biomass']['classes']['solid biomass'].append('Secondary Forestry residues – sawdust')
```

Comment:
Hmm, it's a bit hacky. Maybe add a comment that it is a hack, to be replaced once snakemake is unicode-conform.

Reply:
"hackyness" acknowledged https://github.com/PyPSA/pypsa-eur-sec/pull/88/commits/918d803c0d9b56f7f67898c93b7b7a29a9a88401

Reply:
;-)